egs/aishell/ASR/transducer_stateless/decoder.py

```diff
@@ -95,4 +96,5 @@ def forward(self, y: torch.Tensor, need_pad: bool = True) -> torch.Tensor:
                 assert embeding_out.size(-1) == self.context_size
             embeding_out = self.conv(embeding_out)
             embeding_out = embeding_out.permute(0, 2, 1)
+        embeding_out = self.output_linear(embeding_out)
```

Comment:
Both `self.conv` and `self.output_linear` can be considered as affine transforms.
Do you need to add some non-linear layers between them?